tests/test_prefetch_related.py

```diff
@@ -14,45 +14,40 @@ class Meta:
 
 
 class UserUpdate(generics.UpdateAPIView):
-    queryset = User.objects.exclude(username='exclude').prefetch_related('groups')
+    queryset = User.objects.exclude(username='exclude')
     serializer_class = UserSerializer
+    prefetch_related = ['groups']
```

Comment:
Useful to mention that these are the only meaningful changes in this file. The actual changes within the tests were reducing duplication but otherwise appear to perform the same as previously.

Reply:
I just added one test case to show the differences between with and without prefetch_related queries.